ha/util/mock_health_event_publisher.py

```diff
@@ -0,0 +1,100 @@
+import sys
+import argparse
+
+from cortx.utils.conf_store import Conf
+from ha.util.conf_store import ConftStoreSearch
+
+
+_index = 'cortx'
+cluster_config_file = '/etc/cortx/cluster.conf'
+config_file_format = 'yaml'
+
+def get_data_nodes(conf_store: ConftStoreSearch = None, node_id: str = None) -> list:
+    """
+    Fetches data node ids using HA wrapper class and displays the result
+
+    Args:
+    conf_store: ConftStoreSearch object
+
+    Returns: list of node ids
+    """
+    data_node_ids = conf_store.get_data_pods(_index)
+    print(data_node_ids)
+    return data_node_ids
+
+def get_server_nodes(conf_store: ConftStoreSearch = None, node_id: str = None) -> None:
+    """
+    Fetches server node ids using HA wrapper class and displays the result
+
+    Args:
+    conf_store: ConftStoreSearch object
+    """
+    server_node_ids = conf_store.get_data_pods(_index)
+    print(data_node_ids)
```

Comment:
print(server_node_ids) ??
can we return that instead of print.

Reply:
The aim is to display the output on stdout.

Reply:
ok

Reply:
Gauri is fine with this but she is not able to see the "Resolve Button". Hence I am resolving it on her behalf.